command/cp.go

```diff
@@ -836,3 +860,33 @@ func givenCommand(c *cli.Context) string {
 
 	return cmd
 }
+
+func createExcludeUrls(ctx context.Context, excludes []string, client storage.Storage, srcurls ...*url.URL) ([]*url.URL, error) {
+	result := make([]*url.URL, 0)
+	// fmt.Printf("the source url : %#v\n", srcurl)
+	for _, srcurl := range srcurls {
+		for _, exclude := range excludes {
+			if exclude == "" {
+				continue
+			}
+			sourcePrefix := srcurl.GetUntilPrefix()
+			excludeStringUrl := sourcePrefix + exclude
+			// fmt.Printf("for exclude: %v the url: %v\n", exclude, excludeStringUrl)
+			if !srcurl.IsRemote() {
+				obj, err := client.Stat(ctx, srcurl)
```

Comment:
Could you please add a comment to indicate why the `Stat` call is needed?

Reply:
added the comment for stat call.